test/model/test_store.dart

```diff
@@ -162,13 +169,24 @@ class TestGlobalStore extends GlobalStore with _ApiConnectionsMixin, _DatabaseMi
   /// The given initial snapshot will be used to initialize a corresponding
   /// [PerAccountStore] when [perAccount] is subsequently called for this
   /// account, in particular when a [PerAccountStoreWidget] is mounted.
-  Future<void> add(Account account, InitialSnapshot initialSnapshot) async {
+  ///
+  /// By default, the account is marked as [IntGlobalSetting.lastVisitedAccountId].
```

Comment:
nit: refer to `setLastVisitedAccount` instead, since that's now the interface the bulk of the app consumes